tests/runtime/test_http_server.py

```diff
@@ -229,7 +229,7 @@ def test_inference(sample_task_model_id):
         assert json_response["greeting"] == "Hello world"
 
 
-def test_inference_optional_field(sample_task_model_id):
+def test_inference_sample_task_optional_field(sample_task_model_id):
```

Comment:
should this test change the `inputs` bit to be
`"inputs": {"name": "world"},`
in order to make sure that that bit is correct and isn't causing the error?